recipes/r-shogun/test.R

```diff
@@ -0,0 +1,12 @@
+library('shogun')
+
+X <- matrix(rnorm(300), 100)
+Y <- matrix(rnorm(300), 100) + .5
+
+mmd <- QuadraticTimeMMD()
+mmd$set_p(RealFeatures(X))
```

Comment:
keep it like this, but for the time being this will fail as we have a bug in the R interface :)